src/NServiceBus.RabbitMQ.AcceptanceTests/ConfigureEndpointRabbitMQTransport.cs

```diff
@@ -24,34 +24,62 @@ class ConfigureEndpointRabbitMQTransport : IConfigureEndpointTestExecution
 
     public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings)
     {
-        connectionStringBuilder = new DbConnectionStringBuilder
+        var connectionString = Environment.GetEnvironmentVariable("RabbitMQTransport.ConnectionString");
+
+        if (string.IsNullOrEmpty(connectionString))
         {
-            ConnectionString = Environment.GetEnvironmentVariable("RabbitMQTransport.ConnectionString")
-        };
+            throw new Exception("The 'RabbitMQTransport.ConnectionString' environment variable is not set.");
+        }
 
-        ApplyDefault(connectionStringBuilder, "username", "guest");
-        ApplyDefault(connectionStringBuilder, "password", "guest");
-        ApplyDefault(connectionStringBuilder, "virtualhost", "nsb-rabbitmq-test");
-        ApplyDefault(connectionStringBuilder, "host", "localhost");
+        connectionStringBuilder = new DbConnectionStringBuilder { ConnectionString = connectionString };
 
         configuration.UseTransport<RabbitMQTransport>().ConnectionString(connectionStringBuilder.ConnectionString);
 
         return TaskEx.CompletedTask;
     }
 
-    static void ApplyDefault(DbConnectionStringBuilder builder, string key, string value)
+    public Task Cleanup() => PurgeQueues();
+
+    async Task PurgeQueues()
     {
-        if (!builder.ContainsKey(key))
+        if (connectionStringBuilder == null)
         {
-            builder.Add(key, value);
+            return;
         }
-    }
 
-    public Task Cleanup() => PurgeQueues();
+        var connectionFactory = new ConnectionFactory
+        {
+            AutomaticRecoveryEnabled = true,
+            UseBackgroundThreadsForIO = true,
```

Comment:
The trailing comma here is inconsistent with the same code in the transport tests version. I know we disagree on if it should be there or not, so lets at least make it consistent!